bundle/jsky.app.ot/src/main/java/jsky/app/ot/SplashDialog.java

```diff
@@ -64,6 +65,11 @@ public void windowClosed(WindowEvent e) {
             });
         }
 
+        if (Platform.get() != Platform.osx) {
+            // Set the frame icon, useful for Windows and Linux
+            Resources.setOTFrameIcon(this);
+        }
```

Comment:
Actually this is very important, I'll change the code. For some reason that call crashes the JVM on OSX, on the other hand you don't need it in OSX since it will use the icon on the dmg